src/Microsoft.Kusto.ServiceLayer/LanguageServices/LanguageService.cs

```diff
@@ -259,15 +244,15 @@ public void InitializeService(ServiceHost serviceHost, SqlToolsContext context)
             // serviceHost.SetRequestHandler(ReferencesRequest.Type, HandleReferencesRequest);
             // serviceHost.SetRequestHandler(DocumentHighlightRequest.Type, HandleDocumentHighlightRequest);
 
-            serviceHost.SetRequestHandler(SignatureHelpRequest.Type, HandleSignatureHelpRequest);
+            //serviceHost.SetRequestHandler(SignatureHelpRequest.Type, HandleSignatureHelpRequest);     // Kusto api doesnt support this as of now. Implement it wherever applicable. Hover help is closest to signature help
```

Comment:
Please check with Kusto team if this support is available.

Reply:
Yes, I had checked previously. Currently its not supported.